x/stakeibc/keeper/lsm_test.go

```diff
@@ -262,7 +311,12 @@ func (s *KeeperTestSuite) TestShouldCheckIfValidatorWasSlashed() {
 		s.App.StakeibcKeeper.SetParams(s.Ctx, params)
 
 		validator := types.Validator{SlashQueryProgressTracker: tc.progress}
-		actualShouldQuery := s.App.StakeibcKeeper.ShouldCheckIfValidatorWasSlashed(s.Ctx, validator, tc.stakeAmount)
+		actualShouldQuery := s.App.StakeibcKeeper.ShouldCheckIfValidatorWasSlashed(
```

Comment:
Can we test the case where Bob can give himself a little more slack by liquid staking bit by bit, but really not that much, e.g.
- TVL is 1000
- Bob liquid stake 99, which brings TVL to 1099, no query is issued. 
- Bob liquid stakes 10 more, no query is triggered. Checkpoint math: 99+10 = 109
- Bob liquid stakes 10 more, which issues the query

Reply:
chatted offline, decided to not move forward with this test because the action that triggered the change in TVL is agnostic to this test